cpp/src/components/connectivity.cu

```diff
@@ -27,8 +27,6 @@
 #include <type_traits>
 #include <utilities/graph_utils.cuh>
```

Comment:
May better update this to

```
#include <utilities/graph_utils.cuh>

#include <type_traits>
```

to be consistent with the rest of the cuGraph.

Reply:
Reorganized includes.